webpack.mix.js

```diff
@@ -9,7 +9,8 @@ mix.js("./src/js/utils.js", "./src/_includes/static/js/utils.js");
 mix.js("./src/js/search.js", "./src/_includes/static/js/search.js");
 
 mix.copyDirectory("./src/fonts", "./src/_includes/static/fonts")
-	.copyDirectory("./src/images", "./src/_includes/static/images");
+	.copyDirectory("./src/images", "./src/_includes/static/images")
+	.copyDirectory("./node_modules/infusion", "./src/_includes/static/infusion");
```

Comment:
Rather than copying `infusion` to the root directory, it might make more sense to copy it to `/lib/infusion`? `/lib` indicates for containing all third party libraries.

The same `/lib/infusion` directory structure should be maintained in [`eleventy.config.js`](https://github.com/TeddTech/wecount.inclusivedesign.ca/blob/add-uio/eleventy.config.js#L70) by changing it to  `eleventyConfig.addPassthroughCopy({"src/_includes/static/lib/infusion": "lib/infusion"});`